src/less/button/base/button.less

```diff
@@ -21,6 +21,43 @@ a.fake-btn--wide {
     padding-right: @button-wide-padding-horizontal;
 }
 
+button.btn--tertiary {
+    .customBackgroundColorProperty(button-tertiary-background-color);
+    .customColorProperty(button-tertiary-foreground-color);
+    border: none;
+}
+
+button.btn--tertiary:hover,
+button.btn--tertiary:focus,
+button.btn--tertiary:active {
+    .customBackgroundColorProperty(button-tertiary-hover-background-color);
+    .customColorProperty(button-tertiary-hover-foreground-color);
+}
+
+button.btn--tertiary[disabled],
+button.btn--tertiary[aria-disabled="true"] {
+    .customColorProperty(button-tertiary-disabled-foreground-color);
+
+    &:hover,
+    &:focus {
+        .customColorProperty(button-tertiary-disabled-foreground-color);
+        .customBackgroundColorProperty(button-tertiary-background-color);
+    }
+}
+
+button.btn--tertiary.btn--destructive {
+    .customColorProperty(button-tertiary-destructive-foreground-color);
+}
+
+button.btn--loading {
+    padding: 8px 20px;
+}
+
+button.btn--loading .progress-spinner {
```

Comment:
We can change `btn--loading` to just `btn`, then we can potentially remove the need for a `btn--loading` modifier altogether. This is how we did it in the northstar prototype - although we didn't have the different padding above - that might prove a problem.